src/meta/meta_turtle.c

```diff
@@ -189,11 +189,12 @@ int
 __wt_turtle_init(WT_SESSION_IMPL *session)
 {
 	WT_DECL_RET;
-	char *metaconf;
-	bool exist_backup, exist_incr, exist_isrc, exist_turtle, load;
+	char *metaconf, *value;
```

Comment:
I suggest renaming `value` to `unused_value` to make it clear it is not otherwise used.